cloudbuild.yml

```diff
@@ -3,19 +3,8 @@ steps:
   args:
     - functions
     - deploy
-    - hello_http
+    - catch_quatrics_requests
     - --region=us-central1
-    - --source=./helloworld
+    - --source=./qualtrics_server
     - --trigger-http
     - --allow-unauthenticated
```

Comment:
Does this trigger stores the request into filestore?

Reply:
nope not yet. It only prints out what you pass to it for now so you can test your qualtrics stuff. I will implement pubsub next